netket/graph/lattice.py

```diff
@@ -246,6 +316,119 @@ def atoms_coord(self):
         """
         return self._atoms_coord
 
+    @property
+    def coords(self):
+        """
+        Returns list of coordinates of lattice points
+        """
+
+        return self._coords
+
+    def translation_perm(self):
+        perms = []
+        for vec in self.basis_vectors:
+            perm = []
+            for coord in self._coords:
+                hash_coord = coord.copy() + vec
+                hash_coord = _np.matmul(hash_coord, self._inv_dims) % 1
+                # make sure 1 and 0 are treated the same
+                hash_coord = hash_coord - hash_coord // (1 - cutoff_tol)
+                hash_coord = (
+                    _np.power(10, tol_digits) * _np.around(hash_coord, tol_digits)
+                ).astype(int)
+                hash_coord = hash(hash_coord.tobytes())
+                perm.append(self._hash_positions[hash_coord])
+
+            perms.append(tuple(perm))
+        return tuple(perms)
+
+    def rotation_perm(self, period, axes=[0, 1]):
+        perm = []
+        axes = list(axes)
+        rot_mat = _np.zeros([2, 2])
+        rot_mat[0, 0] = _np.cos(2 * pi / period)
+        rot_mat[1, 0] = -_np.sin(2 * pi / period)
+        rot_mat[0, 1] = _np.sin(2 * pi / period)
+        rot_mat[1, 1] = _np.cos(2 * pi / period)
+
+        rot_coords = self._coords.copy()
+        rot_coords[:, axes] = _np.matmul(rot_coords[:, axes], rot_mat)
+
+        for hash_coord in rot_coords:
+            hash_coord = _np.matmul(hash_coord, self._inv_dims) % 1
+            hash_coord = hash_coord - hash_coord // (1 - cutoff_tol)
```

Comment:
I suppose you mean
```suggestion
            hash_coord = _np.remainder(hash_coord @ self._inv_dims, 1)
```
dividing with `(1-cutoff_tol)` is particularly odd